client-ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts

```diff
@@ -46,7 +47,8 @@ export class MessagePackHubProtocol implements IHubProtocol {
     }
 
     private createCloseMessage(properties: any[]): HubMessage {
-        if (properties.length !== 2) {
+        // check minimum length to allow protocol to add items to the end of objects in future releases
+        if (properties.length < 2) {
```

Comment:
I'm not a fan of these magic numbers. What do we think about moving them to private named constants? Makes it clearer what they actually are to the reader.